external/tcl/tclCmdAH.c

```diff
@@ -186,7 +186,7 @@ Tcl_CaseObjCmd(dummy, interp, objc, objv)
 	    char msg[100];
 	    
 	    arg = Tcl_GetStringFromObj(armPtr, &argLen);
-	    sprintf(msg, "\n    (\"%.*s\" arm line %d)", argLen, arg,
+	    snprintf(msg, sizeof(msg), sizeof(msg), "\n    (\"%.*s\" arm line %d)", argLen, arg,
```

Comment:
```suggestion
	    snprintf(msg, sizeof(msg), "\n    (\"%.*s\" arm line %d)", argLen, arg,
```

> [line 189 in external/tcl/tclCmdAH.c](https://github.com/delphes/delphes/blob/0f1ccd70a601816e0b6f6876271aa0ec76480aa4/external/tcl/tclCmdAH.c#L189) looks suspicious

Yes, `git grep` error here.